rdrf/rdrf/forms/consent_forms.py

```diff
@@ -69,6 +71,7 @@ def save(self, commit=True):
             # If patient just created line above was erroring
             patient_registries = []
 
+        notify_clinician = self.user == self.patient_model.user
```

Comment:
Drop this check from here.
Let's notify each time a notification is configured. No matter who saves the consents.
Then you can also drop passing in the `user` everywhere.

Reply:
Updated.